File: Service_eHospital_DongNai_A/Implement/BenhNhan_TienCanService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class BenhNhan_TienCanService : BaseService<BenhNhan_TienCan, IBenhNhan_TienCanRepository>
    , IBenhNhan_TienCanService
    {
    private readonly IBenhNhan_TienCanRepository _BenhNhan_TienCanRepository;
    public BenhNhan_TienCanService(IBenhNhan_TienCanRepository BenhNhan_TienCanRepository) : base(BenhNhan_TienCanRepository)
    {
    _BenhNhan_TienCanRepository = BenhNhan_TienCanRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/BenhNhan_TienCan.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { BenhNhan_TienCan } from './BenhNhan_TienCan.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class BenhNhan_TienCanService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'BenhNhan_TienCan_Id', 'BenhNhan_Id', 'TienCan_Id', 'NgayPhatHien', 'TinhTrang', 'GhiChu', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id'];

    List: BenhNhan_TienCan[] | undefined;
    ListFilter: BenhNhan_TienCan[] | undefined;
    FormData!: BenhNhan_TienCan;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "BenhNhan_TienCan";
    }
}

